### DISCUSSION TITLE
Replacing Gnome with Regolith

### DISCUSSION THREAD

**Discussion by brandleesee** [regolith-linux/regolith-desktop#960] [Q&A]
My intention is to replace Gnome. The particular hardware has installed Ubuntu Gnome 22.04. What would be the preferred method? To install

regolith-system-ubuntu (https://regolith-desktop.com/docs/howtos/minimal-ubuntu-install/)

or

regolith-desktop regolith-session-flashback regolith-session-sway regolith-look-lascaille

?

Thank you in advance.

**Comment by kgilmer** [comment]
Hi @brandleesee , is your goal to prevent `gnome-shell` from running in a Regolith sesssion?  If so you can install `lightdm` and select it as the login manager during the installation.  Upon reboot `gnome-shell` should not be loaded when logging in to a Regolith session.

**Comment by brandleesee** [comment]
Thank you for taking the time to look into my inquiry.

My intention is to have a lightweight system that uses tiling windows as the primary interface.

From the documentation, I understand that a base server ubuntu would require the `regolith-system-ubuntu` meta-package. My lack of understanding is when does one install `regolith-desktop regolith-session-flashback regolith-session-sway regolith-look-lascaille` ?

I do not mind Regolith being a sort of sub-system to Gnome as long as the experience is sleek.

Thank you in advance.

**Reply by kgilmer** [reply]
I believe you're referring to this?  https://regolith-desktop.com/docs/howtos/minimal-ubuntu-install/

You should be able to achieve your goals w/ the following:

X11:
```
$ sudo apt install regolith-system-ubuntu regolith-session-flashback regolith-look-lascaille
```

Wayland:
```
$ sudo apt install regolith-system-ubuntu regolith-session-sway regolith-look-lascaille
```

I haven't tested this setup guide recently so let me know if you experience problems :)
